packages/vhd-lib/aliases.integ.js

```diff
@@ -1,66 +1,67 @@
 'use strict'
 
-/* eslint-env jest */
+const { beforeEach, afterEach, describe, it } = require('test')
+const { strict: assert } = require('assert')
 
+const { rimraf } = require('rimraf')
 const tmp = require('tmp')
 const { getSyncedHandler } = require('@xen-orchestra/fs')
 const { Disposable, pFromCallback } = require('promise-toolbox')
-const { rimraf } = require('rimraf')
 
 const { isVhdAlias, resolveVhdAlias } = require('./aliases')
 const { ALIAS_MAX_PATH_LENGTH } = require('./_constants')
 
 let tempDir
 
-jest.setTimeout(60000)
-
-beforeEach(async () => {
-  tempDir = await pFromCallback(cb => tmp.dir(cb))
-})
+describe('Aliases', async () => {
```

Comment:
`describe` + `it` should be readable like a sentence.

Example: `describe('add()', () => it('returns a number', () => {}))` can be read as `add() returns a number` which makes the tests obvious.

In this case, we have something like `Aliases recognises only *.alias.vhd files`. Do you think it's understandable?

Reply:
Perfect!